ring_doorbell/const.py

```diff
@@ -23,6 +23,7 @@ class OAuth:
 
 # timeout for HTTP requests
 TIMEOUT = 5
+DEFAULT_VIDEO_DOWNLOAD_TIMEOUT = 120
```

Comment:
@ZachBenz can we add some documentation string on why you think 120 seconds was a good time? Maybe we can link the issue #171 which will have some information in case we need to revisit this in the future. What do you think?

Reply:
Agreed - I'll add the comments inline.